extensions/mssql/src/deployment/dockerUtils.ts

```diff
@@ -915,43 +917,13 @@ async function getUsedPortsFromContainers(containerIds: string[]): Promise<Set<n
     return usedPorts;
 }
 
-/**
- * Finds a Docker container by checking if its exposed ports match the server name.
- * It inspects each container to find a match with the server name.
- */
-async function findContainerByPort(containerIds: string[], serverName: string): Promise<string> {
-    if (serverName === localhost || serverName === localhostIP) {
-        serverName += `,${defaultPortNumber}`;
-    }
-    for (const id of containerIds) {
-        try {
-            const inspect = await execDockerCommand(COMMANDS.INSPECT_CONTAINER(id));
-            const ports = inspect.match(/"HostPort":\s*"(\d+)"/g);
-
-            if (ports?.some((p) => serverName.includes(p.match(/\d+/)?.[0] || ""))) {
-                const nameMatch = inspect.match(/"Name"\s*:\s*"\/([^"]+)"/);
-                if (nameMatch) return nameMatch[1];
-            }
-        } catch {
-            // skip container if inspection fails
-        }
-    }
-
-    return undefined;
-}
-
 /**
  * Checks if a connection is a Docker container by inspecting the server name.
```

Comment:
Stale doc, can you update it to the latest parameter and add some decription to it?